CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 Sharpbrake Changelog
 ====================
 
+### [v3.0.2][v3.0.2] (April 5, 2017)
+
+* Fix missing notifier info, environment and app version
+  in the notice context.
```

Comment:
Please link to PRs in changelog. Example: https://github.com/airbrake/airbrake-ruby/blame/master/CHANGELOG.md#L25